applications/DEMApplication/CMakeLists.txt

```diff
@@ -113,10 +112,14 @@ set(KRATOS_DEM_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/random_variable.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/piecewise_linear_random_variable.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/discrete_random_variable.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_rolling_friction_model.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_rolling_friction_model_constant_torque.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_rolling_friction_model_viscous_torque.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_rolling_friction_model_bounded.cpp
-
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_global_damping_model.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_global_damping_model_nonviscous.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_global_damping_model_viscous.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/DEM_global_damping_model_viscousforcedependent.cpp
```

Comment:
I wonder how we should name those damping models. For the one you used, it is the same with eq. 18 in [Zhao's work [2014]](https://onlinelibrary.wiley.com/doi/10.1002/nme.4702). However, he said, "To ensure quasi-static condition, a simple local non-viscous damping force [40] is added to dissipate kinetic energy". 
1. I understand that we are using "global damping" to identify it with the "contact viscous damping" we used for CLs. 
2. As Zhao said that's a non-viscous damping force, please check whether you can call it "viscous force dependent". I am confused about your description.

Reply:
True. I changed the model names to what I think is more representative and, maybe, correct.